src/Components/Aspire.OpenAI/AssemblyInfo.cs

```diff
@@ -0,0 +1,11 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.OpenAI;
+using Aspire;
+using Microsoft.Extensions.Hosting;
+
+[assembly: ConfigurationSchema("Aspire:OpenAI", typeof(OpenAISettings))]
+[assembly: ConfigurationSchema("Aspire:OpenAI:ClientOptions", typeof(AspireOpenAIExtensions))]
+
+[assembly: LoggingCategories()]
```

Comment:
Is this necessary? Or does it throw if it isn't specfiied?

Reply:
Removed. Works without it. I just wanted a placeholder to remember there are no logging categories.